<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1055</width>
    <height>593</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="topMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="widget" native="true">
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="CGridSplitter" name="frame_2">
          <widget class="QTableView" name="tabVAbutments">
           <property name="geometry">
            <rect>
             <x>490</x>
             <y>240</y>
             <width>256</width>
             <height>189</height>
            </rect>
           </property>
          </widget>
          <widget class="QTableView" name="tabVImplants">
           <property name="geometry">
            <rect>
             <x>500</x>
             <y>10</y>
             <width>256</width>
             <height>190</height>
            </rect>
           </property>
          </widget>
          <widget class="QTreeView" name="tVImplants">
           <property name="geometry">
            <rect>
             <x>240</x>
             <y>30</y>
             <width>256</width>
             <height>190</height>
            </rect>
           </property>
          </widget>
          <widget class="QTreeView" name="tVAbutments">
           <property name="geometry">
            <rect>
             <x>190</x>
             <y>250</y>
             <width>256</width>
             <height>189</height>
            </rect>
           </property>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="wButtons" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>6</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="pBOK">
             <property name="text">
              <string>OK</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pBCancel">
             <property name="text">
              <string>Cancel</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pB3DModel">
             <property name="toolTip">
              <string>View/Hide  3D Model</string>
             </property>
             <property name="text">
              <string>3D Model  &lt;&lt;</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="wOpenGL" native="true"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="movable">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CGridSplitter</class>
   <extends>QFrame</extends>
   <header location="global">gridsplitter.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
